monai/handlers/clearml_handlers.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from .tensorboard_handlers import TensorBoardImageHandler, TensorBoardStatsHandler
+
+
+class ClearMLHandler:
+    """
+
+    Base class for the handlers to log everything to ClearML WebUI.
+
+    Args:
```

Comment:
For the doc-strings of args, please follow the other code style, for example:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/dataset.py#L81

Thanks.

Reply:
Done !